text/0010-exception-types.md

```diff
@@ -0,0 +1,75 @@
+* Start Date: 2022-09-17
+* RFC Type: feature
+* RFC PR: 0010
+* RFC Status: draft
+
+# Summary
+
+This RFC suggests a feature which introduces additional types of exceptions next to `unhandled`.
```

Comment:
```suggestion
This RFC suggests a feature which introduces additional types of exceptions next to `mechanism.handled`.
```

Flag `unhandled` does not exist so it's confusing, I've updated the description and issue's title as well.